domains/pallets/messenger/src/fees.rs

```diff
@@ -161,6 +161,17 @@ impl<T: Config> Pallet<T> {
         }
 
         if !inbox_fees.is_zero() {
+            InboxFeesOnHold::<T>::mutate(|inbox_fees_on_hold| {
+                *inbox_fees_on_hold = inbox_fees_on_hold
+                    .checked_sub(&inbox_fees)
```

Comment:
wouldn't the `checked_sub` fails on taurus after migration previous messages wont have the fees held ?

Reply:
A migration is added (and I tested in local dev network) to ensure this won't happen.